lib/src/mockserver/server.ts

```diff
@@ -66,6 +72,10 @@ export function runMockServer(
   };
 }
 
+function stripDraft(url: string): string {
+  return url.replace("/_draft/", "/");
```

Comment:
Now that I look at this, it might be easier to read and less likely to get out of sync with the `if()` that comes above if it were inlined?

Reply:
that's a good point, was taking this out in a separate method for the sake of 'only doing 1 thing'. But I guess it doesn't need it for now as it doesn't have any reusing, I'd rather take out the word `/_draft/` to a separate constant that can be reused anywhere as that's what's happening right now. There are `/_draft` used on the openapi3.ts and here.